baram/iam_manager.py

```diff
@@ -61,3 +62,38 @@ def list_group_policies(self, user_group_name, max_items=100):
         :return:
         '''
         return self.cli.list_group_policies(GroupName=user_group_name, MaxItems=max_items)
+
+    def get_policies(self, scope='All', max_result=1000):
+        """
+        Lists all policies in IAM
+        :param scope: 'Local' for customer managed policies, 'AWS' for AWS managed policies, 'All' for all policies
+        :param max_result: max number of results (max=1000)
+        :return:
+        """
+        policies = self.cli.list_policies(Scope=scope, MaxItems=max_result)
+        result = [i for i in policies['Policies']]
+        if policies['IsTruncated']:
+            marker = policies['Marker']
+            while True:
+                next_policies = self.cli.list_policies(Scope=scope, MaxItems=max_result, Marker=marker)
+                result += next_policies['Policies']
+                if 'Marker' in next_policies:
+                    marker = next_policies['Marker']
+                else:
+                    break
+        else:
+            return result
+        return result
+
+    def get_redundant_policies(self, scope='Local'):
+        """
+        Lists redundant policies that are not attached to any IAM user, group, or role
+        :param scope: 'Local' for customer managed policies, 'AWS' for AWS managed policies, 'All' for all policies
+        :return:
+        """
+        redundant_policies = [i for i in self.get_policies(scope=scope) if i['AttachmentCount'] == 0]
+        return redundant_policies
```

Comment:
These could be shortened to `return [i for i in self.get_policies(scope=scope) if i['AttachmentCount'] == 0]`

Reply:
as you said there will be no variable needed to store values. will reflect in next commit